sparta/src/Scoreboard.cpp

```diff
@@ -314,6 +314,10 @@ namespace sparta
         // we could bind to a Scoreboard in another CPU!  That'd be
         // bad.
         auto cpu_node = parent->findAncestorByName("core*");
+        // if a core node is not available, search from the root
+        if(cpu_node == nullptr){
+            cpu_node = parent->getRoot();
+        }
```

Comment:
Hey Aaron, thanks for the fix.  I think you can remove the query to find the `findAncestorByName` call completely.  Just start with the `parent` node in the search (i.e. remove the concept of `cpu_node` completely).  See if that works.

Reply:
If we are in a multicore system, wouldn't this run into the issue of multiple scoreboards being found?

Reply:
Technically the current way has this problem.  I think just removing the initial path check will work and possibly keep the matches to the core the view is contained in.